src/nvim/channel.c

```diff
@@ -422,7 +424,7 @@ uint64_t channel_connect(bool tcp, const char *address,
     rstream_start(&channel->stream.socket, on_socket_output, channel);
   }
 
-  channel_create_event(channel, NULL);
+  channel_create_event(channel, address);
```

Comment:
included address in log message

Reply:
I though of adding the address in the channel info, but this is good for now :+1: